src/kirby/components/list/list.component.ts

```diff
@@ -119,22 +139,6 @@ export class ListComponent implements OnChanges {
     this.isLoadOnDemandEnabled = this.loadOnDemand.observers.length > 0;
   }
 
-  private getItemOrder(item: any): { isFirst: boolean; isLast: boolean } {
-    const defaultOrder = { isFirst: false, isLast: false };
-    if (!item) {
-      return defaultOrder;
-    }
-    if (!this.isSectionsEnabled) {
-      return defaultOrder;
-    }
-    const order = this.orderMap.get(item);
-    if (!order) {
-      console.warn('Order of list item within section not found!');
-      return defaultOrder;
-    }
-    return order;
-  }
-
   isFirstItem(item: any, index: number) {
```

Comment:
Not related to this issue/PR 😄  But `isFirstItem` + `isLastItem` + `getItemOrder` + `orderMap` can be deleted:
```suggestion
```